networkx/algorithms/centrality/betweenness.py

```diff
@@ -213,7 +212,7 @@ def edge_betweenness_centrality(G, k=None, normalized=True, weight=None, seed=No
 
     Notes
     -----
-    The algorithm is from Ulrik Brandes [1]_.
+    The algorithm is from Ulrik Brandes [2]_.
```

Comment:
Why do you think this is the wrong reference. Both references are by Brandes. I think this is the reference to 1). The reference to 2) is above.

Reply:
Sounds good! Done in ca8dcb1.